wire-ios-sync-engine/Tests/Source/UserSession/ZMUserSessionTests+AccessToken.swift

```diff
@@ -39,14 +41,14 @@ class ZMUserSessionTests_AccessToken: ZMUserSessionTestsBase {
         shouldRenew: Bool
     ) {
         // given
-        var previousApiVersion = BackendInfo.apiVersion
+        let previousApiVersion = BackendInfo.apiVersion
         defer {
             BackendInfo.apiVersion = previousApiVersion
             transportSession.renewAccessTokenCalls = []
         }
         BackendInfo.apiVersion = apiVersion
 
-        let userClient = UserClient.insertNewObject(in: uiMOC)
```

Comment:
@netbe @David-Henner Does this need to be uiMOC or syncMOC?

Reply:
Unfortunately the `userClient` is accessed along with other entities of the `syncMOC` in
```
sut.didRegisterSelfUserClient(userClient)
```